adagrams/game.py

```diff
@@ -1,11 +1,124 @@
+from random import choice
+
+
 def draw_letters():
-    pass
+
+    MAX_LETTERS_IN_HAND = 10
+
+    LETTER_POOL = {
+        'A': 9,
+        'B': 2,
+        'C': 2,
+        'D': 4,
+        'E': 12,
+        'F': 2,
+        'G': 3,
+        'H': 2,
+        'I': 9,
+        'J': 1,
+        'K': 1,
+        'L': 4,
+        'M': 2,
+        'N': 6,
+        'O': 8,
+        'P': 2,
+        'Q': 1,
+        'R': 6,
+        'S': 4,
+        'T': 6,
+        'U': 4,
+        'V': 2,
+        'W': 2,
+        'X': 1,
+        'Y': 2,
+        'Z': 1
+    }
+    
+    letter_pool_list = []
+
+    # Creating a letter pool list to emulate a pile of letters to choose from:
+    for letter in LETTER_POOL:
```

Comment:
Nice job with this, it works well to ensure the distribution of letters according to how many times they can be selected!